packages/core/integration-tests/test/scope-hoisting.js

```diff
@@ -1190,7 +1217,7 @@ describe('scope hoisting', function() {
     });
 
     describe('tree shaking dynamic imports', function() {
-      it('supports tree shaking statically analyzable dynamic import: destructued await assignment', async function() {
+      it.skip('supports tree shaking statically analyzable dynamic import: destructued await assignment', async function() {
```

Comment:
Was there a reason you didn't implement this?
(= we shouldn't support it as with `let fs; fs = require("fs"); fs.readFileSync`)

Reply:
Yeah basically

Reply:
Should we just remove the tests then?